### PR TITLE
bug - use of deprecated syntax (print and except) fixed

### PR DESCRIPTION
**Pull request by carocad**
I am using networkx library to hold a syntantic analysis of python source code and I noticed that you had some deprecated python statements (print and except for Python:v2.5) that could be easily solve.
The new syntaxt shouldn't change anything of the library but mainly for the documentation

Hope it helps ;)

### CODE REVIEW

**Comment by chebee7i** [comment]
Thanks.

**Comment by debugger22** [comment]
Perhaps this change was not tested. Imports from `__future__` should be at the top(first python command). Failing to do so results in a syntax error.

File: doc/make_gallery.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 link_template = """\
 <a href="%s"><img src="%s" border="0" alt="%s"/></a>
 """
-
+from __future__ import print_function
```
Comment (debugger22) [inline doc/make_gallery.py]: This one should be moved to the top i.e. before `template = """\`.

**Comment by chebee7i** [comment]
Good catch. Indeed, `doc/make_gallery.py` does not get covered by the unit tests.

**Comment by debugger22** [comment]
@chebee7i  I wonder if doc generation is covered on travis-ci.

**Comment by chebee7i** [comment]
Fixed in 965640e7399c669980243d8b162c4521339f294f. I don't believe the doc generation is covered.

**Pull request merged**